swarm/runtime/statsdb/ingestion.py

```
@@ -49,6 +50,73 @@ def _insert_raw_event(self, event: Dict[str, Any]) -> bool:
+                        kind VARCHAR,
+                        flow_key VARCHAR,
+                        step_id VARCHAR,
+                        agent_key VARCHAR,
+                        payload JSON
+                    )
+                """
+                )
+
+                # Bulk insert into temp table
+                conn.executemany(
+                    f"INSERT INTO {temp_table} VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)", data
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)